tools/refguide_check.py

```diff
@@ -344,8 +343,8 @@ def is_deprecated(f):
     """
     Check if module `f` is deprecated
 
-    Parameter
-    ---------
+    Parameters
+    ----------
```

Comment:
This should have been caught by the sphinx build, but this file is not part of the docs so sphinx doesn't parse the docstring. I guess `runtests.py` should be better documented.

Reply:
Yeah, but even if caught by sphinx build nothing beats a tool that autofix the typos :-)